client/interledger/js/components/assets.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import { Row, Glyphicon } from 'react-bootstrap/lib';
+
+import Spinner from '../../../lib/js/react/components/spinner';
+
+
+const Assets = React.createClass({
```

Comment:
This seems like a really unnecessary component... is it going to do more in the future?